install/dataloggers.csv

```diff
@@ -899,6 +899,7 @@ Nanometrics,Centaur CTR4-6S,10235,Horomatangi Reefs,,2023-05-25T16:19:00Z,9999-0
 Nanometrics,Centaur CTR4-6S,10236,Tauhara South,,2023-09-20T03:08:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,10840,Waikawa Valley,,2024-03-03T20:30:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,10891,Summer Hill,,2024-07-01T03:10:00Z,9999-01-01T00:00:00Z
+Nanometrics,Centaur CTR4-6S,10894,South Ngauruhow,,2024-08-15T01:40:00Z,9999-01-01T00:00:00Z
```

Comment:
- just a quick tweak on the station name: `Ngauruhow` to `Ngauruhoe` (not an easy place name to spell, I know), thanks @Thomas-Benson 😄